ensime_shared/ensime.py

```diff
@@ -23,6 +23,9 @@
 else:
     from Queue import Queue
 
+# Dictionary mapping a shorthand key to a vim command string, for various
```

Comment:
Isn't this obvious? Don't get me wrong, the rest of the docs are great, but I find these lines really cumbersome.